ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-tech-support.j2

```diff
@@ -0,0 +1,40 @@
+{# Management Tech-Support #}
+{% if management_tech_support is arista.avd.defined %}
+
+## Management Tech-Support
+
+{%   if management_tech_support.policy_show_tech_support is arista.avd.defined %}
+### Policy
+
+{%     if management_tech_support.policy_show_tech_support.exclude_commands is arista.avd.defined %}
+#### Exclude Commands
+
+| Command | Type |
+| ------- | ---- |
+{%       if management_tech_support.policy_show_tech_support.exclude_commands is arista.avd.defined %}
+{%         for exclude_command in management_tech_support.policy_show_tech_support.exclude_commands %}
+| {{ exclude_command.command }} | {{ exclude_command.type | arista.avd.default('text') }} |
+{%         endfor %}
+{%       endif %}
+{%     endif %}
+
+{%     if management_tech_support.policy_show_tech_support.include_commands is arista.avd.defined %}
+#### Include Commands
+
+| Command | Type |
+| ------- | ---- |
+{%       if management_tech_support.policy_show_tech_support.include_commands is arista.avd.defined %}
+{%         for include_command in management_tech_support.policy_show_tech_support.include_commands %}
+| {{ include_command.command }} | {{ include_command.type | arista.avd.default('text') }} |
```

Comment:
In the tests I did on eos, I did not have the json option for include commands. So the type key can be removed for include. Please check on your devices before changing.

Reply:
Well on 7280SR2K I have an option to specify JSON.

Reply:
You are correct - will change it